build/ci/release.yml

```diff
@@ -34,6 +34,22 @@ variables:
     working_dir: src/github.com/mongodb/atlas-cli-plugin-kubernetes
     env:
       <<: *go_env
+  - &silkbomb_container_config
+      SILKBOMB_REPO: "release-infrastructure/silkbomb"
+      SILKBOMB_TAG: "2.0"
+      SILKBOMB_REGISTRY: "901841024863.dkr.ecr.us-east-1.amazonaws.com"
+      SILKBOMB_IMAGE: "901841024863.dkr.ecr.us-east-1.amazonaws.com/release-infrastructure/silkbomb:2.0"
+      SILKBOMB_PURLS_FILE: "/pwd/purls.txt"
+      SILKBOMB_SBOM_FILE: "/pwd/sbom.json"
+  - &kondukto_config
+      KONDUKTO_CREDENTIALS_FILE: kondukto_credentials.env
+      KONDUKTO_REPO: atlascli-plugin-kubernetes
+      KONDUKTO_BRANCH: test
+  - &silkbomb_options
+    working_dir: src/github.com/mongodb/atlas-cli-plugin-kubernetes/build/package
+    include_expansions_in_env: [AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, AWS_SESSION_TOKEN]
+    env:
+      <<: [*silkbomb_container_config, *kondukto_config]
```

Comment:
I think this is a bit too convoluted, I would skip the env vars from here and have only 3 separate anchors:
- `&silkbomb_container_config`
- `&kondukto_config`
- `&silkbomb_options`

I noticed that `silkbomb_options` always includes `KONDUKTO_CREDENTIALS_FILE` but that is ONLY generated for `upload_sbom` not `generate_sbom` so I would set the env vars that are actually used where they are actually required and set the options on their own.

WDYT?

Reply:
I do not see that as a problem. I am not sure if Evergreen allows you to concatenate, but having such flexibility only pays off if you have several registries yo where you send the same image, or the same image with several tags, etc.

for now I would not worry, because [YAGNI](https://medium.com/better-programming/yagni-you-aint-gonna-need-it-f9a178cd8e1) "You Ain’t Gonna Need It"